tests/test_box_utilities.py

```diff
@@ -0,0 +1,82 @@
+# test olio box_utilities functions
+
+import pytest
+import numpy as np
+
+import resqpy.olio.box_utilities as bx
+
+
+def test_box_utilities():
```

Comment:
I think this function would be a lot more understandable if it were broken out into individual unit tests with descriptive names. It is hard for me to understand what the intent of this test is from the code.

Rather than one 70-line test function with no commentary, could it be 10 7-line unit tests?

Reply:
I've added comments for now. If you think breaking it into smaller functions is important I could do that, and simply have a common function for generating the tiny test data that is common to most of the tests. In my opinion these are quite simple tests of quite simple functionality, so I'm not sure it's worth putting much effort into.

Reply:
Those comments help a lot, thanks!